src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -1,482 +0,0 @@
-#!/usr/bin/python
```

Comment:
I don't think we should remove this entire file in this PR as it contains the the *old* `MLContext` that was in the previous release.  I would just remove the `mllearn` and `mlpipeline` stuff from it, as that was added after the last official release.